docs/getting-started.rst

```diff
@@ -262,5 +262,28 @@ add the new lines.
 
 Now, when you press the space bar, projectiles appear. They only appear once
 each time we press the space bar (unless your keyboard supports key repeats,
-then you'll see it shooting every so often). Now we need something to hit with
-our projectiles!
\ No newline at end of file
+then you'll see it shooting every so often). Next we need something to hit with
```

Comment:
Is keyboard repeating something PyGame does not filter for us?

Reply:
If it's happening at the hardware level, I don't think it can tell? Most keyboards don't have it enabled by default, but I've seen it appear in weird cases.

Reply:
It's easy to filter out. Don't allow another key press event until a matching key up event.

Do we have any idea what configurations this happens in? Define "weird"?

Reply:
Specifically, some keyboards don't do "hold" they throw new keypress events on a repeat. Some accessibility tools do it as well. I would rather we _don't_ filter it so we don't end up with our stuff shutting out accessibility features.

Reply:
That particular discussion is a tangent, although one we should maybe discuss?

I would like to just drop the parenthetical, on the grounds that it's better to be accessible than 100% correct. But I also don't know the first thing about writing tutorials, so feel free to ignore me.

Reply:
Mostly want it there for the folks who might have a weird setup for any reason, but we can drop it and when someone encounters it, I know how to put it back.